fairseq/model_parallel/models/pipeline_parallel_transformer/model.py

```diff
@@ -441,7 +441,6 @@ def convert_to_pipeline_parallel_state_dict(self, state_dict):
                 # fmt: off
                 if isinstance(module, TransformerEncoderEmbedding):
                     new_state_dict[f'model.partitions.{pid}.{mid}.embed_tokens.weight'] = state_dict['encoder.embed_tokens.weight']
-                    new_state_dict[f'model.partitions.{pid}.{mid}.embed_positions._float_tensor'] = state_dict['encoder.embed_positions._float_tensor']
```

Comment:
How come this is removed?

Reply:
The `_float_tensor` buffer was just a dummy tensor to track whether the `SinusoidalPositionalEmbedding` object is on CPU or CUDA. Now the `weights` buffer can be moved between CPU and CUDA, we no longer need `_float_tensor`.